src/main/scala/com/typesafe/sbt/SbtProguard.scala

```diff
@@ -55,8 +55,16 @@ object SbtProguard extends AutoPlugin {
     autoImport.proguard := proguardTask.value
   )
 
+  private def groupId(proguardVersionStr: String): String =
+    "^(\\d+)\\.".r
+      .findFirstMatchIn(proguardVersionStr)
+      .map(_.group(1).toInt) match {
```

Comment:
I think sbt has some built-in version parsing now, although maybe not available for sbt 0.13 if this plugin is still cross-compiled.